src/Command/ModelCommand.php

```diff
@@ -819,13 +812,15 @@ public function getValidation(Table $model, array $associations, Arguments $args
      * @param string $fieldName Name of field to be validated.
      * @param array $metaData metadata for field
      * @param array<string> $primaryKey The primary key field. Unused because PK validation is skipped
+     * @param bool $isForeignKey Indicates if the currently precessed field is a foreign key
      * @return array Array of validation for the field.
      */
     public function fieldValidation(
         TableSchemaInterface $schema,
         string $fieldName,
         array $metaData,
-        array $primaryKey
+        array $primaryKey,
+        bool $isForeignKey = false
```

Comment:
This is again change in method signature and will break people's overridden method, me being one of them.

Personally I can simply update my code for the new minor but we shouldn't break our backwards compatibility promise for minor versions.